webapp/src/component/layout/Notifications/TaskAssignedItem.tsx

```diff
@@ -0,0 +1,49 @@
+import { default as React, FunctionComponent } from 'react';
+import { Box, styled } from '@mui/material';
+import { T } from '@tolgee/react';
+import {
+  NotificationItem,
+  NotificationItemProps,
+} from 'tg.component/layout/Notifications/NotificationItem';
+
+const LinkedDetailItem = styled(Box)`
+  margin-right: 10px;
+  display: inline;
+`;
+
+const LinkedDetailNumber = styled(LinkedDetailItem)`
+  color: ${({ theme }) =>
+    theme.palette.mode === 'light'
+      ? theme.palette.emphasis[400]
+      : theme.palette.emphasis[600]};
+`;
+
+type TaskAssignedItemProps = NotificationItemProps;
+
+export const TaskAssignedItem: FunctionComponent<TaskAssignedItemProps> = ({
+  notification,
+  ...props
+}) => {
+  const destinationUrl = `/projects/${notification.project?.id}/task?number=${notification.linkedTask?.number}`;
```

Comment:
Urls shouldn't be constructed in this way. Use the `links.tsx` file to ensure there there are not broken urls in case something gets renamed. Links to tasks are a bit more complicated, but there is already a function for getting the link `getTaskUrl` in ee module